### DISCUSSION TITLE
Shift on home row for alphas

### DISCUSSION THREAD

**Discussion by rcurran1221** [urob/zmk-config#46] [Q&A] [answered]
Anyone experiment with tweaking this config for using shift on the home row for alphas?

**Comment by urob** [comment, marked as answer]
@infused-kim did experiment with that, reducing the tapping term and disabling `hold-trigger-on-release` for shift. In addition you could also try reducing the `prior-idle` term, depending on your typing speed.

Link to their config: https://github.com/infused-kim/zmk-config/blob/8c2cad1185afb594a502872441f11319433dc5e4/config/includes/behaviours_homerow_mods.dtsi#L51-L80

**Reply by rcurran1221** [reply]
curious, how would hold-trigger-on-release impact false negatives for shifting alphas?

**Reply by urob** [reply]
> curious, how would hold-trigger-on-release impact false negatives for shifting alphas?

I'm not entirely sure to be honest. What it does is that it delays the positional checks until key release. This allows chaining multiple hold-taps on the same hand, and still enforce positional hold-taps. But I don't see a direct link to avoiding false positives when shifting alphas.

There's an indirect relation insomuch the upstream implementation of HTOR only works with the balanced flavor. If one were to try other flavors, one would need to patch ZMK to continue using HTOR (see discussion in the readme).
